presto-hive/src/main/java/io/prestosql/plugin/hive/HiveSessionProperties.java

```diff
@@ -91,6 +91,8 @@
     private static final String QUERY_PARTITION_FILTER_REQUIRED = "query_partition_filter_required";
     private static final String PROJECTION_PUSHDOWN_ENABLED = "projection_pushdown_enabled";
     private static final String PARQUET_OPTIMIZED_WRITER_ENABLED = "parquet_optimized_writer_enabled";
+    private static final String REDIRECTING_TO_ICEBERG_ENABLED = "redirecting_to_iceberg_enabled";
```

Comment:
We could use a single property `redirect_to_iceberg` that is the catalog name. If it's not set, we don't redirect. Thoughts?

Reply:
I think having `REDIRECTING_TO_ICEBERG_ENABLED` can make things clearer to users. They can set it false explicitly if they want to fall back to use the Hive table.